src/app/components/testimonials/Testimonials.jsx

```diff
@@ -0,0 +1,74 @@
+import Link from "next/link";
+import Button from "../common/Button";
+import SocialMediaButtons from "../common/SocialMediaBtn";
+import TestimonialsRatings from "../testimonials/TestimonialsRatings";
```

Comment:
here you exiting the folder and entering it again
I'd change it to
```suggestion
import Ratings from "./Ratings";
```